src/handler-context/Config/s3-config-client.ts

```diff
@@ -61,25 +61,16 @@ const parseConfigFile = async (
   }
 };
 
-const getClient = (): S3Client =>
-  new S3Client({
-    region: "eu-west-2",
-    endpoint: process.env.LOCAL_ENDPOINT,
-  });
+const client = new S3Client({
+  region: "eu-west-2",
```

Comment:
Don't we have the region defined in a central place? If not, it might be a good idea to have it somewhere central and import it from there.

Reply:
I don't think we do but yes that would be a good thing to enshrine

Reply:
Added a new ticket to fix this https://govukverify.atlassian.net/browse/BTM-499